### ISSUE TITLE
Syndi Medborgs and hacked hyposprays

### ISSUE THREAD

**Issue by davethwave** [tgstation/tgstation#36982]
Why in the world does the syndi medborgs only have 2 healing chems and doesn't have anything equivalent to a hacked hypospray...

**Comment by NoKnowledge** [comment]
The reason it only has 2 healing chems is because one of its chems, syndicate nanites, is flat out the basic healing bit of adminordrazine without all the "set (hallucinations, brain damage, stun, knockdown ect.) to 0". It is quite frankly a bit inane how much damage it heals. For referance, its 5 brute burn and toxin, 15 of oxy and clone, and 3 clone per lifetick.

**Comment by ExcessiveUseOfCobblestone** [comment]
do be mindful it only appears (normal circumstances) in nuke ops, which already have a lot of powerful gear to tackle the crew. Getting it during wizard is whatever considering the same staff can turn you into a mouse.

Would be kind of lame if the syndicate assault borg is outclassed by the medical equivalent.

**Comment by factoryman942** [comment]
Yeah, the point of syndie mediborg is as a support / survival helper for an op team. If you want a borg equipped for combat, spend the extra 30TC on an assault borg for some decent dakka.

**Comment by GhostEcho** [comment]
Closed yes but on discussion of the syndicate assault borg, it IS outclassed by the medical borg in pretty much all ways. The cyborg LMG acts more like a shittier stechkin that doesn't shoot rapidly enough, the grenade launcher has only six shots which makes it's usability really restricted to the point you either blow them all too earlier, or never use them because you worry about needing them later, and the esword is meh when melee is a huge risk for borgs, meanwhile the medical borg has, syndi nanites, the cyborg hypospray(I believe?), an energy saw, a medibeam gun, as well as an emag and surgical tools. At the very least, assault borg should get flash protection.
